Gemfile

```diff
@@ -27,7 +27,7 @@ gem 'govuk_frontend_toolkit'
 gem 'govuk_template'
 gem 'govuk_elements_rails'
 gem 'govuk_elements_form_builder', git: 'https://github.com/ministryofjustice/govuk_elements_form_builder.git'
-gem 'registers-ruby-client', git: 'https://github.com/openregister/registers-ruby-client.git', tag: 'v0.4.0'
+gem 'registers-ruby-client', git:'https://github.com/openregister/registers-ruby-client.git',  ref: 'd9055bf52af176275c93e924ce22dcef1212708b'
```

Comment:
Can you just reference `branch: 'master'` and run `bundle update registers-ruby-client`